pkg/integration/integration.go

```diff
@@ -23,6 +23,7 @@ import (
 	k3dCluster "github.com/rancher/k3d/v3/pkg/cluster"
 	"github.com/rancher/k3d/v3/pkg/runtimes"
 	k3d "github.com/rancher/k3d/v3/pkg/types"
+	"github.com/rancher/k3d/v3/version"
```

Comment:
Nitpick: Maybe use k3dVersion to follow the (for all I know, only this file) naming pattern